### PR TITLE
Rem 421

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
ir-replenishment-grps 
not in menu

### CODE REVIEW

File: src/repositories/IVReplenishementRepository.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const service = 'IR.asmx/'
+
+export const IVReplenishementRepository = {
+  ReplenishmentGroups: {
+    qry: service + 'qryGRP',
```
Comment (FatimaSalami) [inline src/repositories/IVReplenishementRepository.js:5]: we should add page right?

File: src/pages/ir-replineshment-grps/index.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+const ReplenishmentGroups = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack } = useWindow()
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50 } = options
+
+    const response = await getRequest({
+      extension: IVReplenishementRepository.ReplenishmentGroups.qry,
```
Comment (FatimaSalami) [inline src/pages/ir-replineshment-grps/index.js:25]: we should use page right?

File: src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useForm } from 'src/hooks/form'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { ControlContext } from 'src/providers/ControlContext'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+import { IVReplenishementRepository } from 'src/repositories/IVReplenishementRepository'
+
+export default function ReplineshmentForm({ labels, maxAccess, recordId, invalidate }) {
```
Comment (FatimaSalami) [inline src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js:17]: should invalidate be passed as prop??

File: src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    },
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      name: yup.string().required(),
+      reference: yup.string().required(),
+      defaultMinQty: yup
+        .number()
+        .nullable()
+        .min(0, 'Must be greater than or equal to 0')
```
Comment (FatimaSalami) [inline src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js:37]: should this be included in labels or keep it hardcoded?
and all other conditions same

Comment (Alinadimmokdad) [reply src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js:37]: those are invisible

File: src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js (hunk 3 above)
Comment (Alinadimmokdad) [reply src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js:17]: yes

File: src/pages/ir-replineshment-grps/index.js (hunk 2 above)
Comment (Alinadimmokdad) [reply src/pages/ir-replineshment-grps/index.js:25]: yes i will

File: src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js (hunk 4 above)
Comment (FatimaSalami) [reply src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js:37]: if the condition is false they should appear i guess!

**Pull request merged**